modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc

```diff
@@ -35,3 +35,8 @@ To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-a
 ====
 You cannot create two Argo CD CRs in the same namespace.
 ====
+
+[WARNING]
+====
+Avoid modifying the `argocd-secret` secret that {gitops-shortname} creates by using external secret management solutions such as the External Secrets Operator or Vault plugins. The `openshift-gitops-operator` manages this secret as part of its core functionality. If you modify this secret externally, it can cause reconciliation conflicts, unpredictable behavior, or disruption of Argo CD instances and {gitops-shortname} workflows. To maintain consistency and reliability, allow the {gitops-shortname} Operator to exclusively manage the `argocd-secret` secret.
```

Comment:
```suggestion
Avoid modifying the `argocd-secret` secret that {gitops-shortname} creates, using external secret management solutions such as the External Secrets Operator or Vault plugins. The `openshift-gitops-operator` manages this secret as part of its core functionality. If you modify this secret externally, it can cause reconciliation conflicts, unpredictable behavior, or disruption of Argo CD instances and {gitops-shortname} workflows. To maintain consistency and reliability, allow the {gitops-shortname} Operator to exclusively manage the `argocd-secret` secret.
```

Reply:
Fixed.